src/layouts/LegalLayout.js

```diff
@@ -36,18 +40,22 @@ const GridWrapper = styled.div`
 const StickyWrapper = styled.div`
   @media (min-width: ${get('breakpoints.md')}) {
     position: sticky;
-    top: calc(98px + 54px);
-  }
-  @media (min-width: ${get('breakpoints.lg')}) {
-    top: calc(88px + 54px);
+    top: calc(
+      var(--header-height-desktop) + 54px
```

Comment:
what represents the `54px` ?

Reply:
It's just an extraneous space in order to not have the sticky menu sticking to the header, but instead being a little bit lower :)